### PR TITLE
2 core get /api/topics

### PR DESCRIPTION
**Pull request by lkeating26**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const { getTopics } = require('./controllers/topics.controller')
+
+const { 
+    handleCustomErrors, 
+    handlePsqlErrors, 
+    handleServerErrors
+    } = require('./errors/index.js')
+
+
+app.use(express.json())
```
Comment (aloush-dev) [inline app.js:6]: This isn't needed yet we are not dealing with any requests.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+app.get('/api/topics', getTopics);
+
+app.use(handleCustomErrors);
+
+app.use(handlePsqlErrors);
+
+app.use(handleServerErrors);
+
```
Comment (aloush-dev) [inline app.js]: Great that you're thinking ahead and you will need these at somepoint but I can only see that one of them would be useful to you so far on this ticket?

**Review by aloush-dev — APPROVED** [review]
Nice job.
Theres quite a bit here that isn't needed for this ticket but great job on everything the ticket is asking for!

File: app.js (hunk 1 above)
Comment (lkeating26) [reply app.js:6]: Hi, I've actually removed these for now! I was struggling for ages to think how to check for the 404 path not found so was trying everything :/ . I nchelped and Katherine jogged my memory about the app.all and /*.

**Pull request merged**